common/utils.py

```diff
@@ -74,30 +75,55 @@ def validate_dict(data_dict, required_keys):
     return True
 
 
-def generate_lambda_response(status_code, message):
-    return {
+def generate_lambda_response(status_code, message, headers=None):
+    response = {
         'statusCode': status_code,
         'body': json.dumps(message),
         'headers': {
             'Content-Type': 'application/json',
             "X-Requested-With": '*',
-            "Access-Control-Allow-Headers": 'Access-Control-Allow-Origin, Content-Type,X-Amz-Date,Authorization,X-Api-Key,x-requested-with',
+            "Access-Control-Allow-Headers": 'Access-Control-Allow-Origin, Content-Type,X-Amz-Date,Authorization,'
```

Comment:
make function generate_lambda_with_cors and withou cors, right now all response allow origin *